barretenberg/sol/src/honk/HonkTypes.sol

```diff
@@ -136,5 +136,9 @@ library Honk {
         // Sumcheck
         Fr[BATCHED_RELATION_PARTIAL_LENGTH][CONST_PROOF_SIZE_LOG_N] sumcheckUnivariates;
         Fr[NUMBER_OF_ENTITIES] sumcheckEvaluations;
+        // Gemini
+        Honk.G1ProofPoint[CONST_PROOF_SIZE_LOG_N] geminiFoldUnivariates;
```

Comment:
you could name these geminiFoldComms and geminiFoldEvals (or exclude gemini from the name altogether)

Reply:
also the commitments should be `CONST_PROOF_SIZE_LOG_N - 1`

Reply:
good catch